File: SubgraphIsomorphism/ktruss/code/matlab/runKtrussBenchmark.m
#!/usr/bin/octave -q

arg_list = argv ();
inputFile = arg_list{1};
if length(arg_list) >= 2
    k = str2num(arg_list{2});
    ktruss(inputFile, k);
else
    k = 3;
    ktruss_max(inputFile, k);
endif
    
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Graph Challenge benchmark
% Developer : Dr. Siddharth Samsi (ssamsi@mit.edu)
%
% MIT
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% (c) <2017> Massachusetts Institute of Technology
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

File: SubgraphIsomorphism/ktruss/code/julia/runKtrussBenchmark.jl
include("ktruss.jl");

input_file = ARGS[1];
if length(ARGS) > 1
	k = parse(Int, ARGS[2]);
	E = ktruss(input_file, k);
else
	k = ktruss_max(input_file);
end

#######################################################
# Graph Challenge Benchmark
# Developer : Dr. Siddharth Samsi (ssamsi@mit.edu)
#
# MIT
#########################################################
# (c) <2017> Massachusetts Institute of Technology
########################################################


